_i18n/en.yml

```diff
@@ -42,47 +43,44 @@ explore:
-  paragraph_two:  when he started to wonder about many possibilities to use technology to inspect congresspeople expenses. When he shared this idea with friends, the project growth in strength in no time and the team was gathered with the goal of making this project happen.
-  paragraph_three: Serenata de Amor Operation was born focused on the audition of reimbursements afforded by the Quota to the Exercise of Parliamentary Activity; this package covers alimentation, transport, lodging and even culture, courses and TV signatures for federal deputies in Brazil.
-  paragraph_four: On the 7 of September the first crowdfunding campaign for the project goes live asking for money to pay for the team work full time during 2 months. In the end the campaign raised 20% more than the original target, summing up more than 25,000.00 USD.
-  paragraph_five: 'We officially started the job in early November and since then the team holds daily meetings and has published tons of fully open source content — that is to say, open for the whole community. This makes part of our guideline: it is more than a technology project, our goal it is to spread information and political conscience. We want to settle a new age of open data and its public access.'
+  paragraph_one: We are a technology project that uses artificial intelligence to audit public accounts and support social control. The idea came from the data scientist <a href="https://twitter.com/irio"> Irio Musskopf </a>, as a way to actively participate in the democratic process, overseeing public spending. When sharing the idea with friends, it didn't take long for the project be spread all over the country, with a team of 10 people and more than 600 volunteers willing to do it.
+  paragraph_two: And then, the Operation Serenade of Love has started, focused on monitoring, with the help of technology, the reimbursements made by the Parliamentary Activity Charge (CEAP) - money that costs food, transportation, lodging and even expenses with culture and TV signatures of parliamentarians.
+  paragraph_three: On September 7, 2016, our first collective financing campaign was alive, which managed to raise approximately R$ 80,000, 30% more than the amount requested, enough to finance the operations of the team for the first 90 days.
+  paragraph_four: Since then, our work, based on information and transparency, has gained space in events and lectures around the world, and has been prominent in the media, being covered by more than 90 journalistics vehicles, including the largest in the country, such as <a href="https://globoplay.globo.com/v/6267741/programa/" target="_blank">Fantastico</a>, <a href="https://globoplay.globo.com/v/5608268/" target="_blank">Jornal Nacional</a> and <a href="http://www1.folha.uol.com.br/poder/2017/01/1852180-jovens-criam-robo-que-monitora-despesas-de-deputados-federais.shtml" target="_blank">Folha de S. Paulo</a>, and a great relevance in the scenario of social control in Brazil.
```

Comment:
So far we have been working based on information and transparency. The importance of our work has been acknowledged in numerous invitations to speak in events all around the world and in numerous appearances in the media – including more than 90 different news outlets, counting Brazilian largest ones, such as <a href="https://globoplay.globo.com/v/6267741/programa/" target="_blank">Fantástico</a>, <a href="https://globoplay.globo.com/v/5608268/" target="_blank">Jornal Nacional</a> and <a href="http://www1.folha.uol.com.br/poder/2017/01/1852180-jovens-criam-robo-que-monitora-despesas-de-deputados-federais.shtml" target="_blank">Folha de S. Paulo</a>. In sum we are afraid we have a considerable relevance in social accountability of public expenses in Brazil.